folly/container/detail/F14Table.h

```diff
@@ -187,6 +187,16 @@ struct StdNodeReplica<
 
 #endif
 
+template <class Container, class Predicate>
+void erase_if_impl(Container& c, Predicate predicate) {
+  for (auto i = c.begin(), last = c.end(); i != last;) {
+    auto prev = i++;
+    if (predicate(*prev)) {
+      c.erase(prev);
+    }
+  }
+};
```

Comment:
We effectively build internally with `-Werror,-Wc++98-compat-extra-semi`, so this extra `;` breaks the build.

Reply:
Fixed. One of my vim plugins added it and I didn't notice; sorry.

Any opposition to a separate MR turning on the extra semicolon warning to the compiler flags for the CMake build?

Reply:
Thanks.

We build with `-Wextra-semi` in many clang builds. Haven't checked if gcc has anything similar.